docs/DeveloperGuide.md

```diff
@@ -296,15 +296,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
+1.  Should work on any mainstream OS as long as it has Java `11` or above installed.
 2.  Should be able to hold up to 1000 expenses/incomes without a noticeable sluggishness in performance for typical usage.
 3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+4.  The data should be stored locally and should be in a human-editable text file.
+5.  The data should not be stored in a Database Management System (DBMS).
+6.  Should not require an installer.
+7.  Should not depend on any remote server.
 
 *{More to be added}*
```

Comment:
Can I confirm that this line will be left unchanged? Otherwise LGTM.

Reply:
Yes, it will remain unchanged.

Reply:
Ok, we will review this again towards the end of v1.2.